Silicon/AlderlakePkg/Include/Library/HeciMeExtLib.h

```diff
@@ -1,7 +1,7 @@
 /** @file
   Heci ME Extended Measured boot library.
 
-  Copyright (c) 2022, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) 2022 - 23, Intel Corporation. All rights reserved.<BR>
```

Comment:
Had better merge the 2 commits into 1 commit given one commit just updates the copy right year.

Reply:
done